jenkins/spark-tests.sh

```diff
@@ -211,6 +211,17 @@ run_iceberg_tests() {
   fi
 }
 
+ # Test spark-avro with documented way of deploying at run time via --packages option from Maven
+run_avro_tests() {
+  unset TEST_PARALLEL  # Enable auto spark local parallel in run_pyspark_from_build.sh
+  export PYSP_TEST_spark_jars_packages="org.apache.spark:spark-avro_2.12:${SPARK_VER}"
+
+  # Workaround to avoid appending avro jar file by '--jars'
+  rm -vf $LOCAL_JAR_PATH/spark-avro*.jar
```

Comment:
Maybe move out of the way (e.g. rename `spark-avro.jar` to `spark-avro.jar%`) before calling the avro tests and restore after to avoid side effects and creating dependence on the order of the tests?

Reply:
refined

Reply:
lets take care of all avro_test inconsistency in https://github.com/NVIDIA/spark-rapids/issues/6532